### PR TITLE
Remove grunt commands from README.md

### PR DESCRIPTION
**Pull request by catrope**
In both repos, running npm install also runs grunt, so it doesn't need
to be run separately.

### CODE REVIEW

**Comment by divec** [comment]
Maybe I'm being stupid, but "npm install" doesn't run grunt for me – is it only the first time you do "npm install"?

**Comment by edg2s** [comment]
It does for me and should according to https://docs.npmjs.com/misc/scripts

> prepare: Run both BEFORE the package is packed and published, on local npm install without any arguments, and when installing git dependencies (See below). This is run AFTER prepublish, but BEFORE prepublishOnly.

Maybe you have an old version of npm?

File: README.md [hunk 1]
```diff
@@ -12,8 +12,8 @@
 - Initialise and build the VE sub-module
 - `cd lib/ve`
 - `npm install`
-- Download, install and run [Apertium-APy](https://github.com/apertium/apertium-apy) .
- (or alternatively, edit `demo/demo.ll.js` and change `http://localhost:2737/translate` to point to a public Apertium-APy instance)
+- `cp environment.json.apertium-example environment.json`
+ (or alternatively, use `environment.json.yandex-example` and put in your Yandex API key)
 - Run a local static webserver in the repository directory: `python3 -m http.server`
```
Comment (edg2s) [inline README.md]: we should probably be less specific here about which web server to use, we shouldn't suggest python3 is a dependency

Comment (catrope) [reply README.md]: OK sure, but my patch doesn't touch that line at all

**Comment by divec** [comment]
We discussed, and decided to decline this because with the old npm in ubuntu 18.04 (npm version 3.5.2), npm install doesn't run grunt for you.

**Pull request closed without merging**